ledger/src/blockstore_processor.rs

```diff
@@ -978,9 +991,11 @@ fn verify_ticks(
     Ok(())
 }
 
+#[allow(clippy::too_many_arguments)]
```

Comment:
These functions as well as ReplayStage are in need of some cleanup. One easy win I think would be packing all the stuff that only gets accessed by the bottom layer of the call-stack (`execute_batch()`) into a struct and unpacking the struct in the bottom layer instead of passing each individual piece in every function.

That being said, I would like to defer that work after this PR - soon enough, work on this project will be lots of experiments / data collection. At that point, I'll have more "idle" time that I could come back to perform this cleanup is there is nothing else more pressing for me

Reply:
> I'll have more "idle" time

you wish